src/core/mark/title.ts

```diff
@@ -67,7 +67,7 @@ export function drawCircularTitle(
     const scaledEndX = padding + txtWidth;
 
     // Determine the points of a rope element for a lebel
-    const ropePoints: number[] = [];
+    const ropePoints: import('pixi.js').Point[] = [];
```

Comment:
incorrect type

Reply:
No there is no difference! It would be more stylistic I guess. Would you have a preference? Another Option which I like is:

```typescript
import type * as PIXI from "pixi.js";

const ropePoints: PIXI.Point[] = [];
```

Reply:
I think in TS code, we should prefer:

```typescript
import type { ... } from "...";
```

compared to inline type imports with `import(...)`. I changed this in the source code.

However, I cannot figure out how to get:

```typescript
import type * as HGC from '@higlass/available-for-plugins';
```

to work correctly, so I have left the inline imports for these expressions.